tutorials/point_cloud_to_mesh.md

```diff
@@ -21,7 +21,7 @@ After installing, open CloudCompare and import your point cloud file by going to
 Depending on the number of points in your point cloud, this could take several minutes.
 Once loaded, you should see the following tunnel section:
 
-![Opening the point cloud](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/point_cloud_to_mesh/cloudcompare2.png)
+![Opening the point cloud](https://raw.githubusercontent.com/gazebosim/gz-sim/gz-sim9/tutorials/files/point_cloud_to_mesh/cloudcompare2.png)
```

Comment:
It would be great if we can switch these to `@image` (`\image`) commands so we don't have to keep doing this.

Reply:
updated to use the image tag. [aa47520](https://github.com/gazebosim/gz-sim/pull/2632/commits/aa47520e6e7a10d045a6e6da3e2ce557653b368d)